app_dart/cloudbuild_app_dart_docker.yaml

```diff
@@ -0,0 +1,17 @@
+# Provide instructions for google Cloud Build to build and upload the flutter
+# dashboard docker image to artifact registry.
+
+steps:
+  # Build dashboard.
+  - name: us-docker.pkg.dev/$PROJECT_ID/flutter/flutter
+    entrypoint: '/bin/bash'
+    args: ['cloud_build/dashboard_build.sh']
```

Comment:
Let's move this step to `cloudbuild_app_dart_deploy.yaml`, as this belongs to the deploy process.

Reply:
@keyonghan I might have misunderstood, but I thought this step was required for setting up the docker image that is deployed in the next step?

Reply:
If you take a look at `cloud_build/dashboard_build.sh`, you can see this step builds the `dashboard` and copies files over which prepares the app_dart deploy.

Reply:
@keyonghan So I moved this step into the deployment and it looks like index.html can't be found: https://version-test-dot-flutter-dashboard.appspot.com/

Reply:
Discussed a bit more with Keyong - added some comments to increase clarity. Resolving!